_posts/2022-08-11-temp-title/my-4-biggest-internship-takeaways.Rmd

```diff
@@ -0,0 +1,50 @@
+
+### 4. Any amount of work you do is appreciated
+
+Going into my internship, I was concerned that I wouldn't be able to do anything right or that I would spend all summer getting close to nothing done. My first week was a bit intimidating because I was the only intern and I did not have as much experience as the rest of my team. However, as the summer went on, I picked up more small tips that were very valuable and helped me excel. It wasn't until my second to last week that I realized that even if I had only completed one task, I still contributed more than I would have had I not been there. <br> <br> One thing I realized about working at this company (as with most) is that the tasks are never ending. Even if you finish one challenging, long task, there are still so many to be done so it's very helpful having another body to help out. Once I came to learn that there was so much work that had to be done, I realized that my work was actually beneficial and not unimportant busy work that can often be assigned to interns. Even when I wasn't physically working and just shadowing others, I got so much insight into how other people work and picked up little skills that I got to incorporate that either made tasks more efficient.
+
+# Final Thoughts
+
+Overall, I'm so grateful I got to work alongside a great group of people this summer and I have definitely grown both as a data scientist and a person, and that is all thanks to them. I have learned so much through their mentor ship and kindness and honestly had such a blast this summer. Work is work but if you're surrounded by the right people, there will never be a dull moment. The knowledge I am taking away and the skills I have gained are immeasurable. I could not have asked for a better internship experience.
```

Comment:
Either here or in a separate Acknowledgements section, I would name/thank everyone who contributed to your internshp experience.